handler/asset_allocations_handler.go

```diff
@@ -1,13 +1,127 @@
 package handler
 
 import (
+	"encoding/json"
+	"fmt"
 	"net/http"
 
+	"github.com/google/uuid"
+	"github.com/gorilla/mux"
+	"github.com/vkhichar/asset-management/contract"
+	"github.com/vkhichar/asset-management/customerrors"
 	"github.com/vkhichar/asset-management/service"
 )
 
-func CreateAssetAllocationHandler(assetAllocation service.AssetAllocationService) http.HandlerFunc {
+func CreateAssetAllocationHandler(assetAllocationService service.AssetAllocationService) http.HandlerFunc {
```

Comment:
rename assetAllocationService to only service

Reply:
In that case it will give a conflict when we try to access *service.Claims later in the function